src/cardboard-button.js

```diff
@@ -4,11 +4,79 @@ import videojs from 'video.js';
 const Button = videojs.getComponent('Button');
 
 class CardboardButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+
+    window.addEventListener('vrdisplayactivate', () => this.handleActivate_());
+    window.addEventListener('vrdisplaydeactivate', () => this.handleDeactivate_());
```

Comment:
scratch that its going to take a revamp of `player.js` which I would rather do in another PR